packages/solo/src/reset-state.js

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 import fs from 'fs';
 
-import { getBestSwingStore } from '../check-lmdb';
+import { getBestSwingStore } from '@agoric/swing-store-lmdb/check-lmdb';
```

Comment:
I don't think `swing-store-lmdb` exposes `check-lmdb`, does it? I don't see that file there on current trunk, or in this PR. If we add that as part of the public API, we should do something to remind ourselves that we've committed to that (I think package.json has a way to enumerate the exports?).